### ISSUE TITLE
Components continually rerender

### ISSUE THREAD

**Issue by illogikal** [formidablejs/formidablejs#13]
Steps to reproduce:

formidable new example-app

```
import { props } from './props'
import { Home } from './pages/Home'
import { About } from './pages/About'

export tag App
	def render
		console.log 'hi'
		<self[ff:Nunito ta:center]>
			<nav[p:7 fs:large]>
				<a[fw:bold] route-to="/$"> "Home"
				" | "
				<a[fw:bold] route-to="/about"> "About"

			<Home route="/$">
			<About route="/about">

```

Console continually displays hi's over and over signifying constant rerenders.

**Comment by illogikal** [comment]
I actually found the problem
export tag Props
    def render
        console.log 'hi'
        <self>
            if isWaiting(props) && hasAttr(self) then props = readProps(self)

        # imba.commit!
this imba commit in the props component is the problem

**Comment by donaldp** [comment]
Hey @illogikal 

Thanks for reporting the issue, a fix will be pushed out as soon as possible, alternatively you can send a PR to https://github.com/formidablejs/view

The file: https://github.com/formidablejs/view/blob/main/formidable/vendor/resources/imba/props.imba

**Comment by donaldp** [comment]
@illogikal issue has been resolved in formidablejs/view:0.1.3 

See [https://github.com/formidablejs/view/pull/5/commits/6d55b99399c6d9565e294d28da8edb30528a62ee](https://github.com/formidablejs/view/pull/5/commits/6d55b99399c6d9565e294d28da8edb30528a62ee)
